extensions/poetry/main.ts

```diff
@@ -0,0 +1,53 @@
+import { PhylumApi } from "phylum"
+import { parse } from "https://deno.land/std/flags/mod.ts"
+
+// Add packages to the lockfile, and upload it to Phylum for analysis.
+// Return `true` if the packages pass the project's thresholds.
+// Return `false` if the packages don't pass the thresholds, or the processing
+// job is not finished.
+async function poetryCheck(args: string[]) {
+    console.log("Updating package lock…")
+
+    let process = Deno.run({ cmd: ["poetry", "add", '--lock', ...args] })
+    await process.status()
+    await process.close()
+
+    console.log("Package lock updated.\n")
```

Comment:
Similar comment on wording here.

Reply:
I'd just remove this log line altogether, honestly. `poetry` already prints messages, the last one being `Writing lockfile`.